test-case/src/main/java/pkg/UsingMarker.java

```diff
@@ -13,8 +14,10 @@
   private final Logger logger = LoggerFactory.getLogger(getClass());
 
   void method() {
-    Marker marker = null;
+    Marker marker = MarkerFactory.getMarker("my marker");
     logger.error(marker, "Hello, marker");
+    logger.error(marker, "Hello, {}");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal "Hello, {}" 3 times. ([squid:S1192](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.findbugs.slf4j%3Afindbugs-slf4j&pullRequest=105&issues=AWZbKQND9tx2us2yCfhg&open=AWZbKQND9tx2us2yCfhg)

[ISSUE_KEY]: AWZbKQND9tx2us2yCfhg